src/common/transport.cc

```diff
@@ -269,6 +269,11 @@ Transport::asyncWriteImpl(Fd fd)
                     wq.push_front(WriteEntry(std::move(deferred), bufferHolder, flags));
                     reactor()->modifyFd(key(), fd, NotifyOn::Read | NotifyOn::Write, Polling::Mode::Edge);
                 }
+                else if (errno == EBADF) {
```

Comment:
Please add a comment referencing the URL for the bug in github.  Code that ignored EBADF looks fishy (or it will in a year or so when someone else is spelunking here).  i will merge PR as soon as you've done this.  TYVM for the work-around.

Reply:
You are very welcome! Happy to be able to give something back :).